openbas-front/src/admin/components/settings/users/Users.js

```diff
@@ -47,7 +47,7 @@ const useStyles = makeStyles(() => ({
 
 const inlineStyles = {
   user_email: {
-    width: '25%',
+    width: '20%',
```

Comment:
Why are you changing this ?

Reply:
I have some tags that were closely spaced in users, so I adjusted the column size for that. However, I could also display fewer tags to avoid the problem. :)